spec/features/user_add_items_to_carts_spec.rb

```diff
@@ -2,27 +2,32 @@
 
 RSpec.feature "UserAddItemsToCarts", type: :feature do
   before(:each) do
-    Food.create(
+    F1 = Food.create(
       name: "Spagetti bolognese",
       description: "Warm yummy spagetti to increase your life span!",
-      price: 1500
-    )
+      price: 1500,
+      prep_time: 12)
+    O1 = Order.create(Status: "Completed", total: 23600,
+                   vat: 2100, delivery_cost: 500, user_id: 6, 
+                   created_at: "2015-10-20 15:49:28", updated_at: "2015-10-20 15:49:28", 
+                   transaction_id: "1VX775486T314881H", invoice: "", pickup_time: 26)
```

Comment:
Line is too long. [85/80]<br>Align the elements of a hash literal if they span more than one line.